src/workers/newNotificationV2Mail.ts

```diff
@@ -78,6 +78,7 @@ export const notificationToTemplateId: Record<NotificationType, string> = {
   squad_public_approved: '41',
   post_bookmark_reminder: '',
   streak_reset_restore: '',
+  squad_featured: '', // awaiting product's response
```

Comment:
Maybe remove for now, or it will block :)

Reply:
Yeah, I will check with the product team if they can provide the details today, if not, I will remove this and write a ticket.

Reply:
Still don't have the ID. Replaced the comment with a TODO towards this ticket: https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-514